python/lsst/ts/ofc/OptCtrlDataDecorator.py

```diff
@@ -162,35 +155,32 @@ def _getRbStroke(self):
 
         return rbStrokeFloat
 
-    def _calcMirrorAuth(self, mirrorDirName, actuatorForceFileName,
-                        usecols=None):
+    def _calcMirrorAuth(self, dofGroup, actuatorForceFileName):
         """Calculate the distribution of control authority of mirror.
 
         This is based on the standard deviation of actuator forces for each
         bending mode is used. The unit is 1 N RMS.
 
+        DOF: degree of freedom.
+
         Parameters
         ----------
-        mirrorDirName : str
-            Mirror directory name.
+        dofGroup : enum 'DofGroup'
+            DOF group.
         actuatorForceFileName : str
-            Mirror actuator force file name
-        usecols : int or sequence, optional
-            Which columns to read, with 0 being the first. For example,
-            usecols = (1,4,5) will extract the 2nd, 5th and 6th columns.
-            The default, None, results in all columns being read.
+            Mirror actuator force file name.
 
         Returns
         -------
         numpy.ndarray
             Authority of mirror.
         """
 
-        filePath = os.path.join(self.configDir, mirrorDirName,
-                                actuatorForceFileName)
-        paramReader = ParamReader(filePath=filePath)
-        mat = paramReader.getMatContent()
-        bendingMode = mat[:, usecols]
+        bendingModeToForce = BendModeToForce()
+        bendingModeToForce.config(self.configDir, dofGroup,
+                                  actuatorForceFileName)
+        bendingMode = bendingModeToForce.getRotMat()
+
         authority = np.std(bendingMode, axis=0)
 
         return authority
```

Comment:
if you change this, consider change to "return np.std(..)!" as well. See above for patch in Iter...

Reply:
Fixed in the commit: a2945e0. Thanks!